### ISSUE TITLE
Addition to audio description definition - Possible errata

### ISSUE THREAD

**Issue by alastc** [w3c/wcag#3254]
The current definition of audio description is:

> narration added to the soundtrack to describe important visual details that cannot be understood from the main soundtrack alone 

The question (started in #1768) is whether an audio track without gaps still requires an audio description.

The suggestion is to make the definition: 

> narration added to the soundtrack, **in pauses of the dialogue**, to describe important visual details that cannot be understood from the main soundtrack alone 

To make it clearer that if there are no pauses in the dialogue it is not applicable.

**Comment by patrickhlauke** [comment]
so the question then is though: if i do have a video, and it has visual information not conveyed by the dialogue BUT also lacks pauses in the dialog... do i then just pass it under the SC? mark the SC as not applicable? the problem (there's no audio description) is still there for the user....

**Comment by patrickhlauke** [comment]
and to me, *this* aspect, if that's really what we're saying, is important to highlight in the understanding document. making it clear that for 1.2.5 in particular, even IF the video would need audio description (because there is visual information not conveyed by the existing audio), it doesn't fail the criterion IF there are no gaps for AD there. but that as a *best practice* authors should strive to provide an extended alternative that makes space for AD

**Comment by alastc** [comment]
> do i then just pass it under the SC? mark the SC as not applicable? the problem (there's no audio description) is still there for the user....

Indeed, I think NA would be appropriate, and it makes logical sense when you consider the AAA version for extending the soundtrack to fit in audio-desc.

Yep, would need an update to the understanding doc to go with it.

**Comment by awkawk** [comment]
This doesn't seem worth changing the normative definition. This can be clarified in Understanding. 

I'm willing to bet that if we asked AD providers how often they've encountered videos have absolutely no space for any AD that they would say close to none. Similarly, if we ask how often the completeness of the description is impacted by the available space, they would probably say most of the time.

If a video includes a single audio description segment, does that meet the SC?

**Comment by yatil** [comment]
FWIW, this is pretty clear [in the WAI guidance](https://www.w3.org/WAI/media/av/planning/#wcag-standard).

The requirements for Audio Description are:
- A: Audio Description in narrative gaps OR Descriptive Transcript
- AA: Audio Description in narrative gaps (if this cannot be done, you already need to have the descriptive transcript from A)
- AAA: Audio Description is provided for a video, even if that means altering the video to fit the description

**Comment by patrickhlauke** [comment]
"pretty clear" except nobody reads that bit of guidance / apocrypha ... they just look at core WCAG, and a cold reading of [1.2.5](https://www.w3.org/WAI/WCAG21/Understanding/audio-description-prerecorded), and even of the definition of [audio description](https://www.w3.org/WAI/WCAG21/Understanding/audio-description-prerecorded#dfn-audio-description), gives no  clear clue about the "unless there are no gaps, in which case this requirement is not applicable" - yes, the definition has the note about "In standard audio description..." but that reads more informative than anything else, there's no explicit hint about what happens when there aren't gaps.

**Comment by detlevhfischer** [comment]
@AWK 
> if we asked AD providers how often they've encountered videos have absolutely no space for any AD that they would say close to none. 

In audits, I get videos with wall-to-wall off-voice, explaining how wonderful a product, region or whatever is, ALL THE TIME.

In my experience, customers then almost never attempt to squeeze in extra AD even if there were a bit of space since it seriously impairs the perceived quality and integrity of the video - it is much more obviously an ugly band-aid type repair than in any other content remediation. The solution, of course, is to graciously embed any significant visual information in the audio at the time of production. (Doesn't happen often, unfortunately.)

**Comment by bruce-usab** [comment]
Does anyone disagree that there is general consensus that a video which has **constant** _conversation_ is pass against SC 1.2.5 1.2.5 Audio Description (Prerecorded) ?

**Comment by yatil** [comment]
@patrickhlauke With “pretty clear” I meant that there is a document that clearly outlines the distinction. I’m 100% for making that distinction clear in the normative text, be it in the Success Criteria itself or in the definition space.

(Not that this is going to happen, as WCAG 2 is considered done by the Working Group.)

**Comment by nayanecom** [comment]
> The current definition of audio description is:
> 
> > narration added to the soundtrack to describe important visual details that cannot be understood from the main soundtrack alone
> 
> The question (started in #1768) is whether an audio track without gaps still requires an audio description.
> 
> The suggestion is to make the definition:
> 
> > narration added to the soundtrack, **in pauses of foreground audio**, to describe important visual details that cannot be understood from the main soundtrack alone
> 
> To make it clearer that if there are no pauses in the dialogue it is not applicable.

As someone who makes films, I think this would be very helpful clarity to have.

**Comment by mraccess77** [comment]
There can also be text at the beginning or end of video that could be spoken but often isn't.  I would think this type of text such as phone numbers, contact details, should be covered since it's at the beginning and end and thus there is room to address it.
